source/administration/downgrade.rst

```diff
@@ -0,0 +1,86 @@
+Location of your local storage directory
+  The local storage directory contains all the files that users have attached to their messages. If you don't know its location, open the System Console and go to **Files > Storage** and read the value in **Local Storage Directory**. Relative paths are relative to the ``mattermost`` directory. For example, if the local storage directory is ``./data/`` then the absolute path is ``{install-path}/mattermost/data``.
+Owner and group of the install directory - *{owner}* and *{group}*
+  Use the ``ls -l {install-path}/mattermost/bin/mattermost`` command to get the owner and group.
+
+#. In a terminal window on the server that hosts Mattermost Server, change to your home directory. If any, delete files and directories that might still exist from a previous download.
+
+   .. code-block:: sh
+
+     cd ~
+
+#. Stop Mattermost Server.
+
+   On Ubuntu 14.04 and RHEL 6.6:
+
+   .. code-block:: sh
+
+     sudo service mattermost stop
+
+   On Ubuntu 16.04 and RHEL 7.1:
+
+   .. code-block:: sh
+
+     sudo systemctl stop mattermost
+     
+
+     
+#. Back up your data and application.
+
+   #. Back up your database using your organization’s standard procedures for backing up MySQL or PostgreSQL.
+
+   #. Back up your application by copying into an archive folder (e.g. ``mattermost-back-YYYY-MM-DD-HH-mm``).
+
+    cd {install-path}/mattermost
+    sudo setcap cap_net_bind_service=+ep ./bin/mattermost
+
+        cd {install-path}
+        sudo cp -ra mattermost/ mattermost-back-$(date +'%F-%H-%M')/
+        
+        
+#. Run the following SQL statements to revert the changes to the database made by the migration. XXXXXX GG Are there commands that have to be run before entering these SQL statements?
+
+```
+UPDATE Teams SET SchemeId = NULL;
+UPDATE Channels SET SchemeId = NULL;
+
+UPDATE TeamMembers SET Roles = CONCAT(Roles, ' team_user'), SchemeUser = NULL where SchemeUser = 1;
+UPDATE TeamMembers SET Roles = CONCAT(Roles, ' team_admin'), SchemeAdmin = NULL where SchemeAdmin = 1;
+
+UPDATE ChannelMembers SET Roles = CONCAT(Roles, ' channel_user'), SchemeUser = NULL where SchemeUser = 1;
+UPDATE ChannelMembers SET Roles = CONCAT(Roles, ' channel_admin'), SchemeAdmin = NULL where SchemeAdmin = 1;
+
+DELETE from Systems WHERE Name = 'migration_advanced_permissions_phase_2';
+```
+
+#. Start Mattermost server.
+
+   On Ubuntu 14.04 and RHEL 6.6:
+
+   .. code-block:: sh
+
+     sudo service mattermost start
+
+   On Ubuntu 16.04 and RHEL 7.1:
+
+   .. code-block:: sh
+
+     sudo systemctl start mattermost
+     
+     
+
+XXXXXX GG any more steps to be added or are applicable based on the upgrade guide? https://docs.mattermost.com/administration/upgrade.html
```

Comment:
I don't think so. Maybe you could note that those SQL commands could take a while to run on large installations. Don't ask me for specific run times - just could conceivably be minutes rather than milliseconds.